pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -91,6 +91,11 @@
                                     Condition="Exists('$(DotNetHostBinDir)/dotnet$(SymbolFileExtension)')">
       <IsSymbolFile>true</IsSymbolFile>
     </ArchitectureSpecificNativeFile>
+    <ArchitectureSpecificNativeFile Include="$(DotNetHostBinDir)/apphost$(ApplicationFileExtension)" />
```

Comment:
i might have misunderstood your comments. I thought you said that we may not version AppHost along with M.N.App. So now how do you decide upon the version of AppHost, do we hard code it with sdk ?

Currently we are using the App host from M.N.App, which has been working well so long, my take away from the other thread was to stick to this